nrt/data/DynaFlow/GeneratorPVTfo/GeneratorPVTfo.par

```diff
@@ -82,19 +82,19 @@
   </macroParameterSet>
   <macroParameterSet id="macro_svc">
     <!-- Generator 1 parameters -->
+    <par type="BOOL" name="secondaryVoltageControl_Participate0_0_" value="true"/>
```

Comment:
I don't get why this has not been changed in the dyd, we still have _1 and _2 instead of _0 and _1.